case_studies/los_altos_high_school/15_09_11_first_meeting/planning.md

```diff
@@ -0,0 +1,15 @@
+#Planning
+
+An outline of what's going to happen at the event
+A list of problems that you anticipate you might run into and how you might solve them
+Any other associated planning done to make that event great.
```

Comment:
@ssun098 Are lines 3-5 supposed to be here? They seem like an explanation of what `planning.md` should be, instead of what you did to plan for the meeting.